index.php

```diff
@@ -51,22 +70,50 @@
             <input type="submit" name="sort" value="COUNTRY">
         </form>
     </div>
-    <form>
-        <input type="text" placeholder="SEARCH...">
-        <input type="submit" value="SEARCH">
+    <form action="index.php" method="GET">
+        <input type="text" id="search" name="search" placeholder="SEARCH...">
+        <input id="submit" type="submit" value="SEARCH">
     </form>
 </nav>
 <main>
     <?php
-    foreach ($result as $item) {
-        echo '<div class="mcflurry">';
+
+    //covers the case where search doesn't return anything
+    if (count($result) === 0){
+        echo "<h3>No Results Found</h3>";
+    }
+    else {
+        //can probably refactor this. Maybe make a function to assign all nulls to unknown
+        //in the functions php file
```

Comment:
Good idea. If you don't do it now, it might never happen, so why not give it a go?

Reply:
have done this now